templates/snippets/shared/icons/star.liquid

```diff
@@ -0,0 +1,9 @@
+{%- default style_class: blank -%}
+{%- default width: blank -%}
+
+<svg class="{{ style_class }}" aria-hidden="true" focusable="false" role="presentation"
+  xmlns="http://www.w3.org/2000/svg" width="17" height="15" viewBox="0 0 16 15" fill="none">
```

Comment:
Let's convert width and height into configurable options. Example:

`{%- default width: blank %}`

`width="{{ width }}"`

Reply:
@aaronryematawaran you should these parameters from the parent template. This will help us to reuse this icon in different contexts.

Reply:
@gersongcg where can I add value to width and height? In the default like `{%- default width: 20 %}`?

Reply:
@aaronryematawaran it should be passed from the template using this snippet.

`{% render 'shared/icons/star', style_class: 'sc-icon-star', width: 17, height: 16 %}`

Reply:
@gersongcg I see, got it! Thanks